packages/angular/generators.json

```diff
@@ -106,10 +106,15 @@
     },
 
     "setup-mfe": {
-      "factory": "./src/generators/setup-mfe/compat",
+      "factory": "./src/generators/setup-mfe/setup-mfe.compat",
       "schema": "./src/generators/setup-mfe/schema.json",
       "description": "Generate a Module Federation configuration for a given Angular application."
     },
+    "scam": {
+      "factory": "./src/generators/scam/scam.compat",
+      "schema": "./src/generators/scam/schema.json",
+      "description": "Generate a component with an accompanying SCAM."
```

Comment:
I wouldn't assume everyone knows what SCAM is. I think it would be better to have something more descriptive here. Maybe we can have something along the lines of:

- `Generates a component with a Single Angular Module Component (SCAM).`
- `Generates a component with a module that only contains and exports the generated component (a.k.a. SCAM).`

Wdyt?

Reply:
Sure! :)